.teamcity/settings.kts

```diff
@@ -0,0 +1,30 @@
+package _Self.buildTypes
+
+import jetbrains.buildServer.configs.kotlin.v2018_2.*
+import jetbrains.buildServer.configs.kotlin.v2018_2.buildFeatures.perfmon
+import jetbrains.buildServer.configs.kotlin.v2018_2.triggers.vcs
+
+object Build : BuildType({
+    name = "Build"
+
+    vcs {
+        root(DslContext.settingsRoot)
+    }
+
+    steps {
+        step {
+            name = "Run MATLAB tests"
+            type = "matlabTestRunner"
+            param("logLoggingLevel", "Terse")
+            param("pdfTestArtifact", "artifacts/testResults.pdf")
+            param("sourceFolders", "code")
+            param("runTestParallel", "true")
+            param("filterTestFolderByName", "tests")
+            param("logOutputDetail", "None")
+            param("strict", "true")
+            param("tapTestArtifact", "artifacts/tspResults.tap")
```

Comment:
We can leave most of these at the defaults and possibly only mention the 
`param("sourceFolders", "code")
`to be consistent with the other platforms in this repo like Jenkins
https://github.com/mathworks/ci-configuration-examples/blob/main/Jenkinsfile

Reply:
Yeah, I think at few examples in .md files its bit extended example : Azure DevOps hence added artifacts. I could remove them